test/imp/Makefile

```diff
@@ -2,3 +2,4 @@ DEF = imp
 KORE_MODULE = IMP
 GOLDEN += sum-save-proofs-spec.k.save-proofs.kore.golden
 include $(CURDIR)/../include.mk
+.PHONY: run-repl-script
```

Comment:
I don't think this works :disappointed:. I'll try to figure out what's going on with the repl-script make rule in a follow-up PR.

Reply:
Please no `.PHONY` targets in the tests; these have to be re-run every time you `make test`. File targets are only rebuilt when the test fails or `kore-exec` is updated, which is necessary when iterating through many broken tests to fix them.

Reply:
This was an attempt to solve a different issue, related to the `run-repl-script` Makefile rule I added yesterday. There's something not working as intended which I need to figure out.